packages/legend-graph/src/graphManager/AbstractPureGraphManager.ts

```diff
@@ -76,6 +76,10 @@ export interface GraphBuilderOptions {
   TEMPORARY__keepSectionIndex?: boolean;
   // when we change our handling of section index, we should be able to get rid of this flag.
   TEMPORARY__disableRawLambdaResolver?: boolean;
+  // skip specific post processing used when editing graph. i.e `freezing` a generated/dependency element.
+  TEMPORARY_skipGraphBuilderPostProcessing?: boolean;
+  // skip post processing specific to generated elements
+  TEMPORARY_skipGeneratedElementsPostProcessing?: boolean;
```

Comment:
This is fine for now, I think in the future, we can aim to refactor `GraphBuilderInput` to remove the `ParentElement` thing. Instead, we could derive the generating element of elements by using an index we keep in `GenerationModel`, that's where the info should be.